doc/source/configuration/wazuh.rst

```diff
@@ -0,0 +1,435 @@
+
+.. code-block:: console
+
+  [wazuh-manager]
+    <wazuh.vm.name>
+
+
+Manually provisioned VM
+-----------------------
+
+In case where you can’t use infra-vms to deploy your wazuh-manager but you want to configure
+host using kayobe, there are some tips (note that depending on your setup this don’t have to always apply):
+
+* Depending on preferences host have to be part of some group in inventory. ``infra vm`` group still seems as best choice
+  You can use ``kayobe infra vm host configure`` to configure host in this case.
+  Bellow tips are based on assumption that infra-vm will be used.
+* user ``stack`` with password less sudo and accessible with ssh keys needs to be present on host.
+  It can be achieved in many different ways, depending on your setup.
+* lvm configuration should be placed in ``host_vars/<host_name>``
+* wazuh-manager host have to be part of ``infra-vms`` group (directly or as child)
+* network used on host needs to be defined in ``networks.yml`` and
+  if you have pre-alocated IP, it should be added to ``network-allocation.yml``. For example:
+
+
+``networks.yml``:
+
+.. code-block:: console
+
+    undercloud_admin_net_cidr: 10.10.224.0/24
+    undercloud_admin_net_allocation_pool_start: 10.10.224.3
+    undercloud_admin_net_allocation_pool_end: 10.10.224.200
+    undercloud_admin_net_gateway: 10.10.224.254
+
+
+``network-allocation.yml``:
+
+.. code-block:: console
+
+    undercloud_admin_net_ips:
+      nesmetprd01: 10.10.224.5
+
+
+Deploying Wazuh Manager services
+================================
+
+Setup
+================================
+
+To install specific version modify wazuh-ansible entry in ``etc/kayobe/ansible/requirements.yml``:
+
+.. code-block:: console
+
+  roles:
+    - name: wazuh-ansible
+      src: https://github.com/stackhpc/wazuh-ansible
+      version: stackhpc
+
+Version above was tested and verified, but there is no reason to use not different one.
+
+Install the role:
+
+``kayobe control host bootstrap``
+
+You might wish to add the following to .gitignore in kayobe-config:
+
+.. code-block:: console
+
+  # Wazuh
+  etc/kayobe/ansible/roles/wazuh-ansible/
+  etc/kayobe/ansible/vars/certificates/*
+  etc/kayobe/ansible/vars/certificates/custom_certificates/*
+
+
+Edit the playbook and variables to your needs:
+
+Wazuh manager configuration
+---------------------------
+
+Wazuh manager playbook is located in ``etc/kayobe/ansible/wazuh-manager.yml``.
+Running this playbook will:
+
+* generate certificates for wazuh-manager
+* setup and deploy filebeat on wazuh-manager vm
+* setup and deploy wazuh-indexer on wazuh-manager vm
+* setup and deploy wazuh-manager on wazuh-manager vm
+* setup and deploy wazuh-dashboard on wazuh-manager vm
+* copy certificates over to wazuh-manager vm
+
+Wazuh manager variables file is located in ``etc/kayobe/inventory/group_vars/wazuh-manager/wazuh-manager``.
+
+You may need to modify some of the variables, including:
+
+* domain_name
+* wazuh_manager_ip
+* private_ip
```

Comment:
I don't see this one in the config

Reply:
Nice catch. Deleted along with domain_name